app/src/main/res/layout/empty_view_layout.xml

```diff
@@ -1,36 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:orientation="vertical"
-              android:gravity="center"
-              android:layout_centerInParent="true"
-              android:paddingLeft="40dp"
-              android:paddingRight="40dp"
-              xmlns:tools="http://schemas.android.com/tools">
+<LinearLayout

```

Comment:
There are no actual code changes here, right? Why did you change the file? If it is exactly the same as the `develop` version, CI will not complain.